### PR TITLE
Fix staticcheck failures of test/integration/client test/integration/disruption

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix staticcheck failures of test/integration/client test/integration/disruption

**Which issue(s) this PR fixes**:

Ref #81657

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/integration/disruption/disruption_test.go [hunk 1]
```diff
@@ -163,6 +163,9 @@ func TestPDBWithScaleSubresource(t *testing.T) {
 	waitPDBStable(t, clientSet, 4, nsName, pdb.Name)
 
 	newPdb, err := clientSet.PolicyV1beta1().PodDisruptionBudgets(nsName).Get(pdb.Name, metav1.GetOptions{})
+	if err != nil {
+		t.Error(err)
```
Comment (neolit123) [inline test/integration/disruption/disruption_test.go]: > t.Errorf("Error getting PodDisruptionBudget: %v", err)

makes it easier to search and to match the error handling of the `Create` call above (L159).

File: test/integration/disruption/disruption_test.go [hunk 2]
```diff
@@ -163,6 +163,9 @@ func TestPDBWithScaleSubresource(t *testing.T) {
 	waitPDBStable(t, clientSet, 4, nsName, pdb.Name)
 
 	newPdb, err := clientSet.PolicyV1beta1().PodDisruptionBudgets(nsName).Get(pdb.Name, metav1.GetOptions{})
+	if err != nil {
+		t.Errorf("Error getting PodDisruptionBudget: %v", err)
+	}
```
Comment (oomichi) [inline test/integration/disruption/disruption_test.go]: Nice addition.

**Review by oomichi — COMMENTED** [review]
/lgtm

**Pull request merged**